### ISSUE TITLE
How can I close the grule debug log

### ISSUE THREAD

**Issue by kelvinkuo** [hyperjumptech/grule-rule-engine#125]
**Is your feature request related to a problem? Please describe.**
I can't find a way to close the grule debug log, its a little noisy

**Describe the solution you'd like**
supply a api to set the log level or shut down the log

**Comment by jinagamvasubabu** [comment]
you can discard the logs using `logrus. SetOutput(ioutil.Discard)`

I know disabling the logs is not an ideal option, this is one of the options to fix this for now

**Comment by newm4n** [comment]
Or, you can increase the log level, to... eg. Fatal, 
In your app, 

```go
import (
    "github.com/sirupsen/logrus"
)

...
...

logrus.SetLevel(logrus.FatalLevel)
```

**Comment by kelvinkuo** [comment]
Thanks, but I don't want it affects my own log level, I use logrus in gin framework too.
@newm4n @jinagamvasubabu

**Comment by newm4n** [comment]
I've made a pull request for this. #130

**Comment by kelvinkuo** [comment]
> I've made a pull request for this. #130

waiting for a new tag

**Comment by newm4n** [comment]
Included in new release v1.7.0
